include/mbgl/storage/default_file_source.hpp

```diff
@@ -98,7 +100,7 @@ class DefaultFileSource : public FileSource {
      * executed on the database thread; it is the responsibility of the SDK bindings
      * to re-execute a user-provided callback on the main thread.
      */
-    void deleteOfflineRegion(OfflineRegion&&, std::function<void (std::exception_ptr)>);
```

Comment:
I could not find a clang-format option that does this. Even `SpaceBeforeParens: Always` does not insert them.